cmd/analyze/main.go

```diff
@@ -103,6 +103,12 @@ func dynamicAnalysis(pkg *pkgecosystem.Pkg) {
 		if err != nil {
 			log.Fatal("Failed to upload file write analysis results to blobstore", "error", err)
 		}
+		for _, writeBuffer := range results.FileWriteBuffers {
+			writeBufferErr := resultstore.New(bucket, resultstore.BasePath(path)).SaveWriteBuffer(ctx, pkg, writeBuffer, utils.GetSHA256Hash(writeBuffer))
```

Comment:
When I run the analysis locally, I will sometimes see a lot of .tmp files. I'm not sure why that is the case? This also does generate a lot of files. Do we want to make storing write buffer files off by default in the local analysis?

Reply:
Not sure if we need to make writing buffers off by default. How big does the local directory get?

Also, do these .tmp files stick around, are they temporary. It's possible this is just some underlying implementation of the gocloud filesystem store.

Reply:
Taking a quick look at one of the runs I get a folder of 48.7 kB. Hm, sometimes I see them and sometimes I don't so I'm not sure if they stick around. I'll take a second look. I do not get these .tmp files on minio though so you may be right.

Reply:
Sounds good. If this is indeed just a temporary artifact, seems OK to me. 

How big does the local directory get though?

Reply:
By "how big does the local directory get" do you mean how large is the tmp/writeResults folder? Just ran the local analysis once on the Django package I'm seeing 13.4MB.

Reply:
I may have ran it for a different package to get 48.7kB above.